vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -150,12 +149,12 @@ def _get_schema(self, weight_quant: BaseModel,
 
         if self.quant_format == CompressionFormat.int_quantized.value:
             if self._is_static_tensor_w8a8(weight_quant, input_quant):
-                return CompressedTensorsW8A8StaticTensor(
-                    strategy=weight_quant.strategy)
+                return CompressedTensorsW8A8(strategy=weight_quant.strategy,
+                                             input_dynamic=False)
 
             if self._is_dynamic_token_w8a8(weight_quant, input_quant):
-                return CompressedTensorsW8A8DynamicToken(
-                    strategy=weight_quant.strategy)
+                return CompressedTensorsW8A8(strategy=weight_quant.strategy,
+                                             input_dynamic=True)
 
```

Comment:
nit: rename `input_dynamic` to `is_static_quant` maybe ? (just for clarity) 
and then we can lose the double `if` statement and just do 
```
                return CompressedTensorsW8A8(strategy=weight_quant.strategy,
                                             is_static_quant=self._is_static_tensor_w8a8(weight_quant, input_quant))
```